packages/react-scripts/fixtures/kitchensink/integration/env.test.js

```diff
@@ -1,33 +1,46 @@
-import { expect } from 'chai'
-import initDOM from './initDOM'
+import { expect } from 'chai';
+import initDOM from './initDOM';
 
 describe('Integration', () => {
   describe('Environment variables', () => {
     it('file env variables', async () => {
-      const doc = await initDOM('file-env-variables')
+      const doc = await initDOM('file-env-variables');
 
-      expect(doc.getElementById('feature-file-env-variables').textContent).to.equal('fromtheenvfile.')
-    })
+      expect(
+        doc.getElementById('feature-file-env-variables').textContent,
```

Comment:
This will crash Node LTS by the way. Need to keep an eye on https://github.com/jlongster/prettier/pull/641.